src/applications/combined-debt-portal/combined/components/DisasterAlert.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const DisasterAlert = () => {
+  return (
+    <va-alert-expandable
+      trigger="Need help with VA Debt after a natural disaster?"
+      status="info"
+    >
+      <ul>
+        <li>
+          <strong>For help with VA benefit debt:</strong> Contact our Debt
+          Management Center by calling <va-telephone contact="8008270648" /> (or{' '}
+          <va-telephone contact="6127136415" international /> from overseas) to
+          request temporary financial relief or via Ask VA at{' '}
+          <a href="https://ask.va.gov">https://ask.va.gov</a> (select "Veterans
+          Affairs-Debt" as the category). We’re here Monday through Friday, 7:30
+          a.m. to 7:00 p.m. ET. If you have hearing loss, call TTY:{' '}
+          <va-telephone contact="711" />.
```

Comment:
You can [add the `tty` prop](https://design.va.gov/storybook/?path=/docs/components-va-telephone--tty) for these kinds of uses.

![Screenshot of the TTY story in storybook with the tty prop highlighted. The rendered markup reads "TTY: 877-345-9876"](https://user-images.githubusercontent.com/2008881/193356940-c1346ac0-55a0-42a6-958c-73d9e852bfcd.png)

Reply:
@bkjohnson Oh neat! I'll update to include that, just missed this after merge 😅